tests/acceptance/content.feature

```diff
@@ -0,0 +1,37 @@
+Feature: Content
+  In order to manage content articles in the web
+  As an owner
+  I need to be able to create and modify content articles
+
+  Background:
+    Given I am logged in at Joomla administrator as "admin" with password "admin"
+    And I open administrator dashboard page
+
+  Scenario: create featured content
+    Given there is a content article with title "My first Joomla Content"
```

Comment:
I agree with Niels previous comment. according to your escenario, `Given that the "article" already exist` you will be "featuring the article" and not "creating a featured article". In the scenario we are not testing the article creation, we asume the article exist, we just test the `feature` feature.